lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize(dishes)
+    @dishes = dishes
+  end
+
+def initialize
+  @dishes = {"patties" => 2.95,
+            "ackee" => 6.95}
+
+  # def print
+  #   dishes.map do |title, price|
+  #     # .map applies function to ever item of iterable and returns a list of results.
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)